Jenkinsfile.integration

```diff
@@ -129,7 +129,10 @@ pipeline {
                 stage('Deploy CaaSP4') {
```

Comment:
This commit (bac2ee2cb93ea68a64c9b2e7c2368af6cd9e8ea0) is not needed given that you switch to the script from skuba in the next commit. Please remove the commit

Reply:
:+1: